src/index.ts

```diff
@@ -1,13 +1,51 @@
 import Router from './router';
 import shorten from './handlers/shorten';
+import list from './handlers/list';
+import deletelink from './handlers/deletelink';
+
+import {
+	validSlackRequest,
+} from './utils';
 
 addEventListener('fetch', event => {
 	event.respondWith(handleRequest(event.request));
 });
 
+async function route(request: Request) {
+	try {
+		if (!(await validSlackRequest(request))) {
+			throw new Error('Request did not come from Slack.');
+		}
+
+		const formData = await request.formData();
+		const text = formData.get('text');
+
+		if (typeof text === 'string') {
+			const args = text.split(" ");
+
+			if (args.length == 1 && args[0] == "") {
```

Comment:
Could there also be a quick comment for what the arguments are supposed to be in this method, or documented as a tooltip somewhere? I noticed it below, but I think a general note in the README would be useful.